R/tt_compatibility.R

```diff
@@ -528,9 +522,10 @@ setMethod(
   }
 )
 
+#' @param x (`VTableNodeInfo`)\cr `TableTree`, `ElementaryTable`, or `TableRow` object.
+#' @param y (`VTableNodeInfo`)\cr `TableTree`, `ElementaryTable`, or `TableRow` object.
+#'
 #' @exportMethod rbind2
-#' @param x `VTableNodeInfo`. `TableTree`, `ElementaryTable` or `TableRow` object.
-#' @param y `VTableNodeInfo`. `TableTree`, `ElementaryTable` or `TableRow` object.
 #' @rdname rbind
 setMethod(
   "rbind2", "VTableNodeInfo",
```

Comment:
this works only for 2? why do we export also this?

Reply:
I'm not really sure what's going on with some of the exporters right now. We probably want to look into this later as well.

Reply:
yes, probably when adding lifecycle #851